Cython/Utility/Exceptions.c

```diff
@@ -761,10 +761,10 @@ static void __Pyx_WriteUnraisable(const char *name, int clineno,
 
 /////////////// CLineInTraceback.proto ///////////////
 
-#ifdef CYTHON_CLINE_IN_TRACEBACK  /* 0 or 1 to disable/enable C line display in tracebacks at C compile time */
-#define __Pyx_CLineForTraceback(tstate, c_line)  (((CYTHON_CLINE_IN_TRACEBACK)) ? c_line : 0)
-#else
+#if CYTHON_CLINE_IN_TRACEBACK_RUNTIME
```

Comment:
Let's make explicitly sure that this evaluates the two macros correctly, however we decide to combine them.
```suggestion
#if CYTHON_CLINE_IN_TRACEBACK && CYTHON_CLINE_IN_TRACEBACK_RUNTIME
```